export const logOut = async () => {
  try {
  } catch (error) {}
};
